.idea/modules/MineInAbyss.iml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module external.linked.project.id="MineInAbyss" external.linked.project.path="$MODULE_DIR$/../.." external.root.project.path="$MODULE_DIR$/../.." external.system.id="GRADLE" external.system.module.group="com.derongan.minecraft" external.system.module.version="1.0-SNAPSHOT" type="JAVA_MODULE" version="4">
```

Comment:
Definitely some module issues between us. Mostly my fault tbh. Ill ignore these files from now on and deal with them later